pkg/cli/build.go

```diff
@@ -229,31 +226,6 @@ func buildCmd() *cobra.Command {
 	return cmd
 }
 
-func getRunner(ctx context.Context, runner string) (container.Runner, error) {
-	if runner != "" {
-		switch runner {
-		case "bubblewrap":
-			return container.BubblewrapRunner(), nil
-		case "docker":
-			return docker.NewRunner(ctx)
-		case "experimentaldagger":
-			return dagger.NewRunner(ctx)
-		default:
-			return nil, fmt.Errorf("unknown runner: %s", runner)
-		}
-	}
-
-	switch runtime.GOOS {
```

Comment:
I think with the new implementation we're missing this, so e2e are failing because we don't have a `default` now, but it expect explicitly to be set